Projects/ChangeTier/Sources/Views/EditDeductibleView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct EditDeductibleView: View {
+    @State var selectedDeductible: String?
+    var vm: SelectTierViewModel
+    @EnvironmentObject var selectTierNavigationVm: ChangeTierNavigationViewModel
+
+    init(
+        vm: SelectTierViewModel
+    ) {
+        self.vm = vm
+    }
+
+    var getDeductibles: [Deductible] {
```

Comment:
This should be let since it doesn't change and it should be initialised in the init